src/system/SystemPacketBuffer.h

```diff
@@ -632,6 +633,44 @@ inline PacketBufferHandle PacketBuffer::Last()
     return PacketBufferHandle::Hold(p);
 }
 
+/**
+ * BufBound backed by packet buffer.
+ */
+class PacketBufBound : public BufBound
+{
+public:
+    /**
+     * Constructs a BufBound that writes into a newly allocated packet buffer.
+     *
+     *  If no memory is available, or if the size requested is too large, then \c IsNull() will be true.
+     *  Otherwise, it is guaranteed that the available space is no less than \a aAvailableSize.
+     *
+     *  @param[in]  aAvailableSize  Minimim octets to allocate after the cursor; see \c PacketBufferHandle::New().
+     *  @param[in]  aReservedSize   Minimum octets to reserve behind the cursor; see \c PacketBufferHandle::New().
+     */
+    PacketBufBound(size_t aAvailableSize, uint16_t aReservedSize = CHIP_SYSTEM_CONFIG_HEADER_RESERVE_SIZE);
+
+    /**
+     * Test whether this PacketBufBound has a backing packet buffer.
+     *
+     * @return \c true if construction failed or if \c Finalize() has been called.
```

Comment:
```suggestion
     * @return \c false if construction failed or if \c Finalize() has been called.
```

Reply:
It's `IsNull` to match `PacketBufferHandle`; given yours and Andy's comments I've taken another crack at the documentation.

Reply:
Ah, yes, I had read the "test whether" bit, not the function name.  :(